app/javascript/packs/forecast.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import Forecast from '../src/components/Forecast/Forecast';
+import { Forecast } from '../src/components/Forecast/Forecast';
```

Comment:
I presume we want components/Forecast/Forecast to return a default Forecast? was this just a got it working thing? is there a reason to not return default? was it just that in the past we could do `export default function` where as now we would need to do 2 lines `const Forecast...` and `export default Forecast;` (I presume my pseudo code works)

Reply:
we sure can change it back to default exports, I just prefer named exports...we don't need two lines now, it's just export const Forecast now. You prefer me to change it back?